README.md

```diff
@@ -24,7 +24,7 @@ It was built for Minitest, but it also has basic support for RSpec and ExUnit.
 1. [Installation](#installation)
 1. [Examples](#examples)
 1. [Usage and Options](#usage-and-options)
-1. [Search Prioty](#search-priority)
+1. [Search Priority](#search-priority)
```

Comment:
I will optimistically merge this, however, we may need to revisit this in the future if feedback indicates pushback to such a controversial change. I'm worried about complaints from power-users who depend on backwards compatibility being maintained, even for the README. Let's move forward for now, but keep an eye out for any pushback.